src/ScottPlot5/ScottPlot/Control/Backend.cs

```diff
@@ -42,6 +42,8 @@ public class Backend
         public event KeyDownHandler KeyDown = delegate { };
         public event KeyUpHandler KeyUp = delegate { };
 
+        public float MinimumDragDistance = 5;
```

Comment:
This should be `const`. And possibly `private`.

Reply:
I think there's real utility for the user to disable this. Personally I don't like it because it makes the plot feel low low FPS when you move it in small circles, when really you're just getting near the mouse origin. 

Maybe there needs to be some logic to track total distance traveled or something? We can pick that up later.